src/Filter/ExceptionTypeFilter.cs

```diff
@@ -0,0 +1,54 @@
+﻿using log4net.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace log4net.Filter
+{
+    public class ExceptionTypeFilter : FilterSkeleton
+    {
+        public ExceptionTypeFilter()
+        {
+            this.AcceptOnMatch = true;
+        }
+
+        /// <summary>
+        /// Set the exception type name to filter
+        /// </summary>
+        public string ExceptionTypeName { get; set; }
+
+        /// <summary>
+        /// <see cref="FilterDecision.Accept"/> when matching <see cref="ExceptionTypeName"/>
```

Comment:
To me the docstring is inconclusive on what this property actually does. It should contain at least a description of what happens when this value is set to "true" and what happens when this value is set to "false". The reasoning is that the documentation is auto-generated to be available online and hence the source of truth for any future reader that would like to use this filter. Future readers should not be required to look at the implementation.